public/locales/hi/translation.json

```diff
@@ -815,7 +815,7 @@
     "title": "सेटिंग्स",
     "general": "सामान्य",
     "actionItemCategories": "कार्य आइटम श्रेणियाँ",
-    "updateOrganization": "संगठन अपडेट करें",
+    "Edit Organization": "संस्था संपादित करें",
```

Comment:
This is not in keeping with the camel case standard. Remove the space, use camel case and update the source code.